zilencer/management/commands/populate_db.py

```diff
@@ -204,6 +203,30 @@ def handle(self, **options):
             zulip_realm_bots.extend(all_realm_bots)
             create_users(realms, zulip_realm_bots, bot=True)
 
+            if options["test_suite"]:
+                # Create test users; the MIT ones are needed to test
+                # the Zephyr mirroring codepaths.
+                testsuite_mit_users = [
+                    ("Fred Sipb (MIT)", "sipbtest@mit.edu"),
+                    ("Athena Consulting Exchange User (MIT)", "starnine@mit.edu"),
+                    ("Esp Classroom (MIT)", "espuser@mit.edu"),
+                    ]
+                create_users(realms, testsuite_mit_users)
+                zulip_realm = Realm.objects.get(domain="zulip.com", name="Zulip Dev")
+                realms = {}
+                for realm in Realm.objects.all():
+                    realms[realm.domain] = realm
+                stream_list = ["Announcement"]
+                create_streams(realms, zulip_realm, stream_list)
+                notifications = []
+                msg = ("Bot created a new stream 'Announcement'. %s"
+                       % (stream_button(stream_list[0])))
+                notifications.append(internal_prep_message(settings.NOTIFICATION_BOT,
+                                   "stream",
+                                   DEFAULT_NOTIFICATION_STREAM_NAME, "Streams", msg,
+                                   realm=zulip_realm))
+                do_send_messages(notifications)
+
```

Comment:
I don't understand why you added this block -- I think all you should have needed to do is create `notifications_stream` above.